test/using_witnet.js

```diff
@@ -97,16 +106,21 @@ contract("Using witnet", accounts => {
       assert(timestamp)
       assert.equal(pkh, accounts[0])
 
+      // Report block
+      blockRelay.postNewBlock(expectedBlockHash, expectedBlockHash, expectedBlockHash, {
+        from: accounts[0],
+      })
+
       // Show PoI of Data Request Inclusion
-      let tx3 = wbi.reportDataRequestInclusion(expectedId, web3.utils.utf8ToHex("PoI"), expectedBlockHash)
+      let tx3 = wbi.reportDataRequestInclusion(expectedId, [drHashRoot], expectedBlockHash, expectedBlockHash)
       await waitForHash(tx3)
 
       let drInfo3 = await wbi.requests(expectedId)
-      let blockHash = drInfo3.drHash
-      assert.equal(expectedBlockHash, blockHash)
+      let DrHash = drInfo3.drHash
+      assert.equal(expectedDrHash, web3.utils.toHex(DrHash))
 
       // Report result
-      let tx4 = wbi.reportResult(expectedId, web3.utils.utf8ToHex("PoI"),
+      let tx4 = wbi.reportResult(expectedId, [1], 1,
```

Comment:
could you use variable names for [1] and 1? It is difficult to know what are you adding